Formula/liblxi.rb

```diff
@@ -0,0 +1,46 @@
+class Liblxi < Formula
+  desc "Simple C API for communicating with LXI compatible instruments"
+  homepage "https://github.com/lxi-tools/liblxi"
+  url "https://github.com/lxi-tools/liblxi/archive/refs/tags/v1.18.tar.gz"
+  sha256 "ebeb8cb51f1024e198b3672aadaba113849d20b26199ae6e4ee679e555e9f274"
+  license "BSD-3-Clause"
+  head "https://github.com/lxi-tools/liblxi.git", branch: "master"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on :macos
+
+  def install
+    # Check if RPC headers are in the system search path
+    sdk_root = `xcrun --show-sdk-path`.chomp
+    rpc_header_path = "#{sdk_root}/usr/include/rpc/rpc.h"
+
+    odie "liblxi requires the macOS RPC framework" unless File.exist?(rpc_header_path)
```

Comment:
Do we need to check for this ourselves? Is this not in `meson.build`?

Reply:
Actually, it looks like pretty much every macOS SDK since at least 10.5, so I think we can drop this.